### PR TITLE
Cameras in use in backend

### PR DESCRIPTION
**Pull request by egedebruin**
You can now get a list of the used cameras and set a camera to used (or not used) in the backend server.

### CODE REVIEW

File: backend/src/main/java/com/benine/backend/camera/BasicCamera.java [hunk 1]
```diff
@@ -73,4 +76,14 @@ public boolean equals(Object obj) {
     }
     return false;
   }
+
+  @Override
+  public boolean isInUse() {
+    return inUse;
+  }
+
+  @Override
```
Comment (Kingdorian) [inline backend/src/main/java/com/benine/backend/camera/BasicCamera.java]: I think it would be clearer to, setInUse, and setNotInUse or something similar.

**Comment by Kingdorian** [comment]
The code looks good and the testing is nice. I think it would be nice however to keep track of who uses a camera but this might be something to add in a next sprint. It would also be nice to have the inuse time out so that it resets if setInUse has not been called for n (5 or so) seconds. This way if a client crashes or something like that it automatically returns to the not in use state after a couple of seconds.

File: backend/src/main/java/com/benine/backend/camera/BasicCamera.java (hunk 1 above)
Comment (egedebruin) [inline backend/src/main/java/com/benine/backend/camera/BasicCamera.java]: Yes I also thought of that as an option, I will change it.

**Comment by Kingdorian** [comment]
We need to merge #144 first.

**Comment by jochem725** [comment]
After #144 has been merged, I agree with merging.

**Pull request merged**